### PR TITLE
OBJ-13 remove screen display filename

### PR DESCRIPTION
**Pull request by mattch1**
Take the user to the "Are you sure you want to remove (attachment name)?" screen

### CODE REVIEW

File: src/controllers/remove.document.controller.ts [hunk 1]
```diff
@@ -1,6 +1,18 @@
+import { Session } from "ch-node-session-handler";
 import { NextFunction, Request, Response } from "express";
 import { Templates } from "../model/template.paths";
+import { Attachment } from "../modules/sdk/objections";
+import * as objectionService from "../services/objection.service";
 
 export const get = async (req: Request, res: Response, next: NextFunction) => {
-  return res.render(Templates.REMOVE_DOCUMENT);
+  const attachments: Attachment[] = await objectionService.getAttachments(req.session as Session);
```
Comment (ICoxCH) [inline src/controllers/remove.document.controller.ts:8]: Don't hate me but in the spec for the api is a {objectionId}/attachments/{attachmentId} endpoint
https://github.com/companieshouse/strike-off-objections-api/blob/master/spec/schema.json#L183
We should probably use that endpoint for this task. But the endpoint doesn't yet exist, if you change this pr against the schema, i'll implement that endpoint

Comment (mattch1) [reply src/controllers/remove.document.controller.ts:8]: Negotiation: Could we keep it as it is for now and I can put a TODO in there stating what you ahve said here?

Comment (mattch1) [reply src/controllers/remove.document.controller.ts:9]: Added the TODO

Comment (mattch1) [reply src/controllers/remove.document.controller.ts]: Removed TODO, adde dnew single attachment service api call.

**Pull request merged**